crates/swc/tests/fixture/blank-space-codegen/strip-only/output/6.ts

```diff
@@ -0,0 +1,4 @@
+(function foo() {}) ();
+
+
+(function bar() {})();
```

Comment:
@magic-akari  Is this what you meant?

Reply:
Yes, for this situation, we may need to retain the parentheses as much as possible. 
Our current operation is to remove the parentheses first and then fix them.
Considering that the type strip does not rely on the AST structure, can we remove the fixer?

Reply:
Yeap, it would be better. I'll patch it soon.

Reply:
Yeap, it would be better. I'll patch it with a follow-up PR. 

Patching it in this PR is possible, but it's better to patch in the next PR because currently there's no `blank_space_mode` in published crates (and in`./bindings`)

Reply:
Maybe I was wrong.
The structure like `(foo?.bar as X).foo` may still need to rely on a fixer when the type is removed.
We need to handle this situation more cautiously.